h/queue.py

```diff
@@ -26,18 +33,22 @@ def get_reader(settings, topic, channel):
     The caller is responsible for adding appropriate `on_message` hooks and
     starting the reader.
     """
+    req.sentry.extra_context(extra_context_data(req, topic))
```

Comment:
Two things:

1. Let's check if `request.sentry` exists before using it. Something like:

    ```python
sentry = getattr(request, 'sentry', None)
if sentry is not None:
        sentry.extra_context(...)
    ```
2. Given we just have the topic and channel, here you can probably just inline this:
  
  ```python
sentry.extra_context({'topic': topic, 'channel': channel})
```